lms/djangoapps/course_api/blocks/api.py

```diff
@@ -71,6 +71,7 @@ def get_blocks(
     transformers = BlockStructureTransformers()
     if requested_fields is None:
         requested_fields = []
+    include_discussions = 'is_discussable' in requested_fields or 'discussion_enabled' in requested_fields
```

Comment:
`is_discussable` and `discussion_enabled` are two completely different things that apply to two completely different blocks, they should not be lumped in together. Only `is_discussable` needs to be considered for this PR.

Reply:
Resolved in next series of commits